setup.py

```diff
@@ -68,6 +68,7 @@
     'test': [
         'black~=19.3b0',
         'flake8~=3.7.5',
+        'flake8~=19.8.0',
```

Comment:
You probably meant `flake8-bugbear` here

Reply:
Truth.